vpr/src/place/place_constraints.h

```diff
@@ -64,4 +64,10 @@ inline bool floorplan_legal(const t_pl_blocks_to_be_moved& blocks_affected) {
  */
 void load_cluster_constraints();
 
+/*
+ * Marks blocks with a region with xlow = xhigh, ylow = yhigh, and subtile specified as fixed
```

Comment:
Make this comment say that it's for when region specifies exactly one x, y, subtile location as legal -> that's when it gets marked as fixed